dashboard/test/helpers/aichat_sagemaker_helper_test.rb

```diff
@@ -0,0 +1,80 @@
+    karen_model_customizations = @common_model_customizations.merge(selectedModelId: AichatSagemakerHelper::MODELS[:KAREN])
+    inputs = AichatSagemakerHelper.format_inputs_for_sagemaker_request(karen_model_customizations, @stored_messages, @new_message)
+    expected_karen_model_inputs = "<|im_start|>system\ntest prompt test retrieval<|im_end|>\n<|im_start|>user\nEdit the following text for spelling and grammar mistakes: new message from user<|im_end|>\n<|im_start|>assistant"
+    assert_equal inputs[:inputs], expected_karen_model_inputs
+  end
+
+  test 'Testing format_inputs_for_sagemaker_request with Arithmo model' do
+    arithmo_model_customizations = @common_model_customizations.merge(selectedModelId: AichatSagemakerHelper::MODELS[:ARITHMO])
+    inputs = AichatSagemakerHelper.format_inputs_for_sagemaker_request(arithmo_model_customizations, @stored_messages, @new_message)
+    expected_arithmo_model_inputs = "Question: test prompt test retrieval new message from user\nAnswer:"
+    assert_equal inputs[:inputs], expected_arithmo_model_inputs
+  end
+
+  test 'Testing format_inputs_for_sagemaker_request with Pirate model for custom stop string' do
+    pirate_model_customizations = @common_model_customizations.merge(selectedModelId: AichatSagemakerHelper::MODELS[:PIRATE])
+    inputs = AichatSagemakerHelper.format_inputs_for_sagemaker_request(pirate_model_customizations, @stored_messages, @new_message)
+    expected_pirate_stop_strings = ["},"]
+    assert_equal inputs[:parameters][:stop], expected_pirate_stop_strings
+  end
+
+  test 'Testing format sagemaker model output for Mistral base model' do
+    model_processor = MistralProcessor.new
+    generated_text = "<s>[INST]where is Chattanooga[/INST] Chattanooga is a city located in the southeastern United States, in the state of Tennessee."
+    model_output = model_processor.format_model_output(generated_text)
+    expected_model_output = " Chattanooga is a city located in the southeastern United States, in the state of Tennessee."
+    assert_equal model_output, expected_model_output
+  end
+
+  test 'Testing format sagemaker model output for Pirate model' do
+    model_processor = PirateProcessor.new
+    generated_text = "<s>[INST]what is your name[/INST] I be a humble servant o' th' briny deep, me hearty, an' I be known by th' name o' th' sea's own whispers. Ye be callin' me Cap'n, an' th' wind be our witness, th' salt be in me veins, an' th' stars be me compass.\" },"
+    model_output = model_processor.format_model_output(generated_text)
+    expected_model_output = " I be a humble servant o' th' briny deep, me hearty, an' I be known by th' name o' th' sea's own whispers. Ye be callin' me Cap'n, an' th' wind be our witness, th' salt be in me veins, an' th' stars be me compass. "
+    assert_equal model_output, expected_model_output
+  end
+
+  test 'Testing format sagemaker model output for Karen model with ' do
```

Comment:
Something missing at end of test name here?

Reply:
Oops! just a typo. Will fix in another quick follow-up which will include Sanchit's suggestion above. Want to merge asap to get a couple small bug fixes in. Thanks for catching!